main.go

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"os"
+
+	"github.com/bitrise-io/go-steputils/v2/stepconf"
+	"github.com/bitrise-io/go-utils/v2/command"
+	"github.com/bitrise-io/go-utils/v2/env"
+	"github.com/bitrise-io/go-utils/v2/log"
+	"github.com/bitrise-steplib/steps-restore-carthage-cache/step"
+)
+
+func main() {
+	os.Exit(run())
+}
+
+func run() int {
+	logger := log.NewLogger()
+	envRepo := env.NewRepository()
+	inputParser := stepconf.NewInputParser(envRepo)
+	cmdFactory := command.NewFactory(envRepo)
+	cacheStep := step.New(logger, inputParser, envRepo, cmdFactory)
```

Comment:
Move step creation to its own function (e.g., https://github.com/bitrise-steplib/steps-fastlane/blob/master/main.go#L56)

Reply:
Does it make sense in such a simple step though? The entire `main.go` file is 30 lines long, so I wouldn't refactor this now, but if the step gets more complex I'm happy to do it. Is that okay?